code/obj/item/organs/head.dm

```diff
@@ -71,16 +71,24 @@
 	disposing()
 		if (holder)
 			holder.head = null
-		skull = null
-		brain = null
-		left_eye = null
-		right_eye = null
-
-		head = null
-		ears = null
-		wear_mask = null
-		glasses = null
-
+		src.skull = null
+		src.brain = null
+		src.left_eye = null
+		src.right_eye = null
+
+		src.head = null
+		if (donor?.head)
+			qdel(donor.head)
```

Comment:
shouldn't this qdel the clothing on the head as well (e.g.: case where head-with-a-hat on the floor gets acid poured on it)

Reply:
head here is the head clothing slot because our clothing naming is insane.